README.md

```diff
@@ -33,6 +33,33 @@ tabrmd on the system bus as this is the most common configuration.
 
 Check out the man page TCTI-TABRMD(7) and TSS2_TCTI_TABRMD_INIT(3).
 
+## tpm2-abrmd vs in-kernel RM
+The current implementations are mostly equivalent with a few differences.
+Both provide isolation between objects & sessions created by different connections which is the core functionality required by applications.
+The reason we have both is that the in-kernel RM was only added very recently and we have TPM2 users in environments with kernels going back to the 3.x series.
+So the user space RM will be around at least till everyone is using the kernel RM.
```

Comment:
I would add that the in-kernel RM was included in 4.12. That way users reading this document will know exactly what are the requirements to use the kernel RM.